dex/src/main/scala/com/wavesplatform/dex/api/websockets/actors/WsInternalClientHandlerActor.scala

```diff
@@ -0,0 +1,131 @@
+package com.wavesplatform.dex.api.websockets.actors
+
+import akka.actor.Cancellable
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.{actor => classic}
+import cats.syntax.option._
+import com.wavesplatform.dex.AssetPairBuilder
+import com.wavesplatform.dex.api.websockets._
+import com.wavesplatform.dex.domain.account.Address
+import com.wavesplatform.dex.domain.asset.AssetPair
+import com.wavesplatform.dex.error.{MatcherError, WsConnectionPongTimeout}
+import com.wavesplatform.dex.time.Time
+
+import scala.collection.immutable.Queue
+import scala.concurrent.duration._
+
+/**
+  * Controls a single WebSocket connection for order updates stream
+  * Handles user messages (pongs) add schedules timeouts (pongs, max connection lifetime)
+  */
+object WsInternalClientHandlerActor {
```

Comment:
Looks like `WsExternalClientHandlerActor` and `WsInternalClientHandlerActor` are pretty similar.. :thinking:

Reply:
There is a common Ping/Pong logic. Had no idea how to share it between typed actors. Also, it is easy for untyped, because we have orElse in partial functions, and we don't care about message type.